modules/nodes-pods-autoscaling-creating-memory.adoc

```diff
@@ -6,9 +6,16 @@
 
 = Creating a horizontal pod autoscaler object for memory utilization by using the CLI
 
-You can create a horizontal pod autoscaler (HPA) for an existing `DeploymentConfig` object or `ReplicationController` object
-that automatically scales the pods associated with that object to maintain the average memory utilization you specify,
-either a direct value or a percentage of requested memory.
+Using the {product-title} CLI, you can create a horizontal pod autoscaler (HPA) to automatically scale an existing 
+`Deployment`, `DeploymentConfig`, `ReplicaSet`, `ReplicationController`, or `StatefulSet` object. The HPA  
+scales the pods associated with that object to maintain the average memory utilization you specify, either a direct value or a percentage 
+of requested memory.
+
+[NOTE]
+====
+It is recommended to to use a Deployment or ReplicaSet objects unless you need a 
```

Comment:
```suggestion
It is recommended to not use a `Deployment` or `ReplicaSet` object unless you need a 
```
(same as the other instance of this note)

Reply:
Same